corehq/apps/smsbillables/management/commands/bootstrap_apposit_gateway.py

```diff
@@ -0,0 +1,42 @@
+from decimal import Decimal
+from django.core.management.base import BaseCommand
+from corehq.apps.accounting.models import Currency
+from corehq.apps.sms.models import INCOMING, OUTGOING
+from corehq.apps.smsbillables.models import SmsGatewayFee, SmsGatewayFeeCriteria
+from corehq.apps.smsbillables.utils import log_smsbillables_info
+from corehq.messaging.smsbackends.apposit.models import SQLAppositBackend
+
+
+def bootstrap_apposit_gateway(apps=None):
+    usd_currency, _ = (apps.get_model('accounting', 'Currency') if apps else Currency).objects.get_or_create(code="USD")
+    sms_gateway_fee_class = apps.get_model('smsbillables', 'SmsGatewayFee') if apps else SmsGatewayFee
+    sms_gateway_fee_criteria_class = apps.get_model('smsbillables', 'SmsGatewayFeeCriteria') if apps else SmsGatewayFeeCriteria
+
+    SmsGatewayFee.create_new(
+        SQLAppositBackend.get_api_id(),
+        INCOMING,
+        Decimal('0.02'),
+        currency=usd_currency,
```

Comment:
@nickpell this is the currency that this rate is quoted in, right? a lot of the other bootstrap gateway commands were using settings.DEFAULT_CURRENCY for this, but it seems like we should pin it to USD if that's the case